packages/js/tsconfig.eslint.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Why do we need this config file? Is `.eslintrc.js` not included by default?

Reply:
My IDE was looking for this file, tried many options to make it not search for this file but could fix it. It's definitely an intellij IDE / eslint plugin problem, but it cannot hurt to add this to fix the problem for now imho